src/backend/src/prisma/schema.prisma

```diff
@@ -555,3 +565,62 @@ model Manufacturer {
   dateDeleted   DateTime?
   materials     Material[]
 }
+
+enum Design_Review_Status {
+  UNCONFIRMED
+  CONFIRMED
+  SCHEDULED
+  DONE
+}
+
+model TeamType {
+  name           String          @id
```

Comment:
can you add an id that's a uuid to this, I've found doing this name/ id thing can get annoying later, you can still make name unique if you want